src/helpers/strategies.ts

```diff
@@ -6,7 +6,8 @@ import { capture } from '@snapshot-labs/snapshot-sentry';
 export let strategies: any[] = [];
 export let strategiesObj: any = {};
 
-const uri = 'https://score.snapshot.org/api/strategies';
+const scoreApiEndpoint = process.env.SCORE_API_URL || 'https://score.snapshot.org';
```

Comment:
Shouldn't this be named `scoreApiURL` like the rest?

Reply:
fixed